src/m365/commands/request.ts

```diff
@@ -0,0 +1,147 @@
+import { AxiosRequestConfig, AxiosRequestHeaders } from 'axios';
+import { Logger } from '../../cli';
+import Command from '../../Command';
+import GlobalOptions from '../../GlobalOptions';
+import request from '../../request';
+import commands from './commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  url: string;
+  method?: string;
+  resource?: string;
+  body?: string;
+  accept?: string;
```

Comment:
Let's remove this option because it's the same as the header name

Reply:
Should I also remove it from the documentation and everywhere else @waldekmastykarz?